tests/unit/App.spec.ts

```diff
@@ -66,7 +69,10 @@ vi.mock('@cornerstonejs/dicom-image-loader', () => ({
   configure: vi.fn(),
   loadImage: vi.fn(),
   webWorkerManager: {
+    // Disable eslint rule for this arrow function
+    /* eslint-disable @typescript-eslint/no-empty-function */
     initialize: vi.fn().mockImplementation(() => {})
```

Comment:
will this work? this should make linter happy
```suggestion
    initialize: vi.fn()
```